skimage/transform/integral.py

```diff
@@ -21,15 +21,28 @@ def integral_image(image):
     S : ndarray
         Integral image/summed area table of same shape as input image.
 
+    Notes
+    -----
+    For better accuracy and to avoid potential overflow, the datatype of the
```

Comment:
```suggestion
    For better accuracy and to avoid potential overflow, the data type of the
```
Not sure, actually, but copying https://scikit-image.org/docs/dev/user_guide/data_types.html. Or "dtype" directly.